pages/search.tsx

```diff
@@ -57,6 +56,24 @@ const SearchPage: NextPage = () => {
     }
   }, [domain]);
 
+  useEffect(() => {
+    if (!identity) {
+      setPpImageUrl("");
+      return;
+    }
+
+    const fetchProfilePic = async () => {
+      try {
+        const imgUrl = await identity.getProfilePic();
+        setPpImageUrl(imgUrl);
+      } catch (error) {
+        setPpImageUrl("");
```

Comment:
Why not using `domain_to_data` simply ? I think the img should be in `domain_to_data` no ?

Reply:
No it's not in `domain_to_data` anymore, from `domain_to_data` we get the contract address of the nft in `verifier_data`, and the nft token id in `extended_verifier_data`